src/client/data/graphql.tsx

```diff
@@ -102,6 +102,8 @@ export enum _MutationKind {
   PublishMany = 'publishMany',
   UnpublishMany = 'unpublishMany',
   DeleteMany = 'deleteMany',
+  SchedulePublish = 'schedulePublish',
+  ScheduleUnpublish = 'scheduleUnpublish',
```

Comment:
I am not sure how these changes came along, I removed my query but must have gotten some new stuff when using codegen. Do I dare remove it or is it okay to leave it? 😅

Reply:
I don't think this should be a problem. It's linked to a change in Giraffe

Reply:
Great, I'll leave it then :D